tests/examples/test_heartbeat.py

```diff
@@ -16,7 +16,7 @@ def test_py(setup_example_test):
 
 @pytest.mark.eda
 @pytest.mark.quick
-@pytest.mark.timeout(300)
+@pytest.mark.timeout(15)
```

Comment:
you can remove this timeout the default is 120s

Reply:
done